### ISSUE TITLE
missalignment with select and icons prefix

### ISSUE THREAD

**Issue by somenxavier** [Dogfalo/materialize#2459]
With this

```
{% extends "base.html" %}
{% block title %}New activity{% endblock %}
{% block content %}
  <script>
  $(document).ready(function() {
    $('select').material_select();
  });
  </script>  

  <h1>New activity</h1>

  <div class="row">
        <div class="input-field col s12">
          <i class="material-icons prefix">account_circle</i>
          <input placeholder="Placeholder" id="identification" type="text" class="validate">
          <label for="indentification">Identification</label>
        </div>
  </div>
  <div class="row">
        <div class="input-field col s12">
          <i class="material-icons prefix">subject</i>
          <textarea id="textarea1" class="materialize-textarea"></textarea>
          <label for="textarea1">Cos</label>
        </div>
  </div>
  <div class="row">
    <div class="input-field col s12">
      <i class="material-icons prefix">subject</i>
      <select id="seleccio" required multiple>
            <option value="1" selected>Option 1</option>
            <option value="2">Option 2</option>
            <option value="3">Option 3</option>
          </select>
          <label for="seleccio">Type</label>
        </div>
  </div>
{% endblock %}
```

I get this:
![captura_2015-12-11_16-54-52](https://cloud.githubusercontent.com/assets/89915/11748345/13c111ba-a028-11e5-948c-5991af75c93e.png)

See that the option 1 is over the icon

**Comment by nathankoop** [comment]
In _form.scss, you can change code (line 150) from

    .prefix ~ input,
    .prefix ~ textarea{
      margin-left: 3rem;
      width: 92%;
      width: calc(100% - 3rem);
    }

to:

    .prefix ~ input,
    .prefix ~ textarea,
    .prefix ~ .select-wrapper {
      margin-left: 3rem;
      width: 92%;
      width: calc(100% - 3rem);
    }

**Comment by MetaiR** [comment]
your code is running perfect but the prefix icon not changing color when click on select element

**Comment by somenxavier** [comment]
@nathankoop please correct in the master version.

**Comment by nathankoop** [comment]
I have sent a PR in (#2479).

If you need it before it get's merged to master then the information is here:
https://github.com/nathankoop/materialize/commit/5063f1e0bba1801fa6f6d7e37d4ae57ec6cebdeb

The two files that need changing are:
* js/forms.js
* sass/components/_form.scss

Then just recompile your JS & SCSS and you're off and running.

There are a couple more things I want to fix in regards to this.

1. The label color for the Select doesn't change when active, unlike the textbox/textarea, but none of the other ones do (radio button, checkbox, etc...) so I want to open a new item and get feedback before I go ahead on it.
2. Javascript code isn't as "nice" as I'd like it to be (could be cleaner, but I wanted to get the working code out first)

**Comment by somenxavier** [comment]
tnanks

**Comment by fega** [comment]
@somenxavier could you check if this is still an issue?

Closed due inactivity, feel free to reopen it if it is still necessary.

**Comment by somenxavier** [comment]
@fega do it yourself. I documented the issue

**Comment by acburst** [comment]
Fixed in 3b8cf2d463db63444e73c8157660379ef3f64c6f
